utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java

```diff
@@ -163,6 +151,39 @@ public void createPullSecret() {
         }
     }
 
+    private void jaegerWorkarounds() {
+        // the syndesis-jaeger doesn't contain "syndesis.io/component" label which is using for finding all components. It is added manually here
+        BooleanSupplier jaegerReady = () -> OpenShiftWaitUtils.isPodReady(
+            OpenShiftUtils.getAnyPod("app.kubernetes.io/instance", "syndesis-jaeger"));
+        new Thread(() -> {
+            int tries = 5;
+            while (tries > 0) {
+                try {
+                    OpenShiftWaitUtils.waitFor(jaegerReady, 30 * 1000);
+                    break;
+                } catch (Exception e) {
+                    ResourceFactory.get(Jaeger.class).ensureImagePull();
+                    log.info("Ensuring jaeger instance is pulled correctly, linking secrets and resetting jaeger pods");
+                }
+                tries--;
+            }
+            try {
+                OpenShiftWaitUtils.waitFor(jaegerReady);
+            } catch (Exception e) {
+                log.error("Jaeger initialization failed ", e);
+                fail("Jaeger initialization failed", e);
```

Comment:
The test cannot fail because in some cases, the Jaeger addon is disabled in the CR.